pegasus/migrations/106_populate_forms_hashed_email.rb

```diff
@@ -0,0 +1,23 @@
+require 'digest/md5'
+
+# Based off of 097_populate_contacts_hashed_email.rb
+Sequel.migration do
+  up do
+    from(:forms).where(hashed_email: nil).each do |form|
+      hashed_email = Digest::MD5.hexdigest(form[:email])
+      from(:forms).
+          where(id: form[:id]).
+          update(hashed_email: hashed_email)
```

Comment:
Our style is indent +2. Though I do personally prefer the +4 indent for line continuations...